EncounterHandler.js

```diff
@@ -689,12 +689,10 @@ function inject_monster_image(stat) {
 		return;
 	}
 	let url = stat.data.largeAvatarUrl;
-	if (url === undefined || url == null || url.length == 0) {
```

Comment:
It's nice that we can drop this. I would also drop the `let url` above, and remove `url !== undefined && url != null && url.length > 0 && ` below. Then you'll need to change the img `src` to reference `stat.data.largeAvatarUrl` as well. 

Sorry to be nitpicky on this, but I think that would clean up this code nicely.

Reply:
no need to apologise at all, I completely agree. I do think initially I added the `stat.data.largeAvatarUrl` as the source then I clocked I still had the let url so removed it